libc-test/build.rs

```diff
@@ -783,6 +784,7 @@ fn test_solarish(target: &str) {
         "sys/mman.h",
         "sys/mount.h",
         "sys/priv.h",
+        "sys/procfs.h",
```

Comment:
I believe this should be `procfs.h`, not `sys/procfs.h`, as per [proc(4)](https://illumos.org/man/4/proc).

Reply:
sure for further mapping that might be better.